__tests__/unit/old-structure/tree-fetching/stratetegies/cache-first.spec.ts

```diff
@@ -73,9 +72,8 @@ describe('Utils: tree fetching (load: cache-first)', () => {
     nock(basePath).get(path).reply(500, JSON.stringify({ error: 'unexpected error' }))
     const onChangeTree = jest.fn()
     await expect(viewClient.load({ url, onChangeTree, strategy, retry })).rejects.toEqual([
-      new BeagleCacheError(url),
       // @ts-ignore
-      new BeagleNetworkError(url),
+      new BeagleNetworkError(url, {} as Response, 500,'GET'),
```

Comment:
```suggestion
      new BeagleNetworkError(url, {} as Response, 500, 'GET'),
```
It would be nice to add the space after the "," to keep a pattern.